ksql-engine/src/test/resources/query-validation-tests/group-by.json

```diff
@@ -0,0 +1,195 @@
+{
+  "comments": [
+    "Tests covering use of the GROUP BY clause"
+  ],
+  "tests": [
+    {
```

Comment:
It would be nice to add a few tests where the `GROUP BY` clause has more than one expression.

Reply:
Agreed.